tests/migrations/test_operations.py

```diff
@@ -575,6 +577,38 @@ def test_create_model_managers(self):
         self.assertIsInstance(managers[2][1], FoodManager)
         self.assertEqual(managers[2][1].args, ("x", "y", 3, 4))
 
+    @skipUnless(connection.vendor == "sqlite", "This is an sqlite specific issue")
+    def test_create_model_with_quote_string(self):
+        operation1 = migrations.CreateModel(
+            "Pony",
+            [
+                ("id", models.AutoField(primary_key=True)),
+                ("pink", models.IntegerField(default=1)),
+            ],
+        )
+        operation2 = migrations.CreateModel(
+            "\"Rider\"",
```

Comment:
You'll want to use `Meta.db_table` here instead to be more realistic. I also don't think multiple models are required to reproduce so try to bring it down to the minimal number of changes required to provide a regression test.